statistics/table.go

```diff
@@ -694,6 +695,28 @@ func (coll *HistColl) GetAvgRowSize(cols []*expression.Column, isEncodedKey bool
 	return size + float64(len(cols))
 }
 
+// GetAvgRowSizeListInDisk computes average row size for given columns.
+func (coll *HistColl) GetAvgRowSizeListInDisk(cols []*expression.Column, padChar bool) (size float64) {
+	if coll.Pseudo || len(coll.Columns) == 0 || coll.Count == 0 {
+		for _, col := range cols {
+			size += float64(chunk.EstimateTypeWidth(padChar, col.GetType()))
```

Comment:
Change `pseudoColSize` in `GetAvgRowSize` to `EstimateTypeWidth` as well?

Reply:
`EstimateTypeWidth` returns memory size in `ListInDisk` while `GetAvgRowSize` returns encoded size in TiKV, I think `pseudoColSize` (aka 8.0) is better.